HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py

```diff
@@ -18,13 +18,137 @@
+hltTiclEGammaSuperClusterProducerUnseeded = ticlEGammaSuperClusterProducer.clone()
+
+from Configuration.ProcessModifiers.ticl_superclustering_dnn_cff import ticl_superclustering_dnn
+ticl_superclustering_dnn.toReplaceWith(_SuperclusteringUnseededSequence, 
+                                       cms.Sequence(
+                                            hltTiclTracksterLinksSuperclusteringDNNUnseeded
+                                            + hltTiclEGammaSuperClusterProducerUnseeded
+                                        )
+)
+ticl_superclustering_dnn.toModify(hltTiclEGammaSuperClusterProducerUnseeded,  
+                                  ticlSuperClusters=cms.InputTag("hltTiclTracksterLinksSuperclusteringDNNUnseeded"),
+                                  ticlTrackstersEM=cms.InputTag("hltTiclTrackstersCLUE3DHigh"),
+                                  layerClusters=cms.InputTag("hltHgcalMergeLayerClusters"))
+
+# Ticl mustache
+from Configuration.ProcessModifiers.ticl_superclustering_mustache_ticl_cff import ticl_superclustering_mustache_ticl
+ticl_superclustering_mustache_ticl.toReplaceWith(_SuperclusteringUnseededSequence, 
+                                       cms.Sequence(
+                                            hltTiclTracksterLinksSuperclusteringMustacheUnseeded
+                                            + hltTiclEGammaSuperClusterProducerUnseeded
+                                        )
+)
+ticl_superclustering_mustache_ticl.toModify(hltTiclEGammaSuperClusterProducerUnseeded, 
+                                            ticlSuperClusters=cms.InputTag("hltTiclTracksterLinksSuperclusteringMustacheUnseeded"),
+                                            ticlTrackstersEM=cms.InputTag("hltTiclTrackstersCLUE3DHigh"),
+                                            layerClusters=cms.InputTag("hltHgcalMergeLayerClusters"))
+
+# Alpaka & DNN
+(alpaka & ticl_superclustering_dnn).toReplaceWith(_HgcalLocalRecoUnseededSequence, 
+                    cms.Sequence(
+                        hltHgcalDigis
+                        + hltHGCalUncalibRecHit
+                        + hltHGCalRecHit+hltParticleFlowRecHitHGC
+                        + hltHgcalSoARecHitsProducer
+                        + hltHgcalSoARecHitsLayerClustersProducer
+                        + hltHgcalSoALayerClustersProducer
+                        + hltHgCalLayerClustersFromSoAProducer
+                        + hltHgcalLayerClustersHSci
+                        + hltHgcalLayerClustersHSi
+                        + hltHgcalMergeLayerClusters
+                        ) 
+)
+(alpaka & ticl_superclustering_dnn).toModify(hltHgcalMergeLayerClusters,
+                                             layerClustersEE = cms.InputTag("hltHgCalLayerClustersFromSoAProducer"),
+                                             time_layerclustersEE = cms.InputTag("hltHgCalLayerClustersFromSoAProducer", "timeLayerCluster"))
```

Comment:
Similar comment as above when compared to lines
https://github.com/cms-sw/cmssw/blob/a85bda57ecb39ceb07427c917c4d4dc63594c9cb/HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py#L46-L48
Similarly applied to 
https://github.com/cms-sw/cmssw/blob/a85bda57ecb39ceb07427c917c4d4dc63594c9cb/HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py#L142-L144

In general the "# Alpaka & DNN" part seems to be duplicating modifications from above, potentially making the maintenance of the code harder. Maybe try to simplify - unless there is some reason for the duplication which I failed to see (potentially having to do with the order the modifiers are applied)?

Reply:
Yes indeed, I'll clean up.